src/test/SmartComponents/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -1,26 +1,173 @@
 import React from 'react';
-import { screen, getByText } from '@testing-library/react';
+import { screen, getByText, waitForElement } from '@testing-library/react';
 import { renderWithReduxRouter } from '../../testUtils';
 import CreateImageWizard from '../../../SmartComponents/CreateImageWizard/CreateImageWizard';
 
-describe('Landing Page', () => {
+function verifyButtons() {
+    // these buttons exist everywhere
+    const next = screen.getByRole('button', { name: /Next/ });
+    const back = screen.getByRole('button', { name: /Back/ });
+    const cancel = screen.getByRole('button', { name: /Cancel/ });
+
+    return [ next, back, cancel ];
+}
+
+function verifyCancelButton(cancel) {
+    cancel.click();
+
+    // this goes back to the landing page
+    waitForElement(
+        () => [
+            screen.getByRole('heading', { name: /Image Builder/ }),
+            screen.getByText('Create a new image'),
+        ],
+        { screen }
+    );
+}
+
+describe('Create Image Wizard', () => {
     beforeEach(() => {
         renderWithReduxRouter(<CreateImageWizard />);
     });
+
     test('renders component', () => {
         // check heading
         screen.getByRole('heading', { name: /Create a new image/ });
 
         // left sidebar navigation
-        const nav = screen.getByRole('navigation');
-        getByText(nav, 'Release');
-        getByText(nav, 'Target environment');
-        getByText(nav, 'Registration');
-        getByText(nav, 'Review');
-
-        // buttons
-        screen.getByRole('button', { name: /Next/ });
+        const sidebar = screen.getByRole('navigation');
+
+        getByText(sidebar, 'Release');
+        getByText(sidebar, 'Target environment');
+        getByText(sidebar, 'Registration');
+        getByText(sidebar, 'Review');
+    });
+});
+
+describe('Step Release', () => {
+    beforeEach(() => {
+        renderWithReduxRouter(<CreateImageWizard />);
+
+        // left sidebar navigation
+        const sidebar = screen.getByRole('navigation');
+        const anchor = getByText(sidebar, 'Release');
+
+        // load from sidebar
+        anchor.click();
+    });
+
+    test('clicking Next loads Target environment', () => {
+        const [ next, , ] = verifyButtons();
+        next.click();
+
+        screen.getByText('Destination');
+        screen.getByText('Secret access key');
+    });
+
+    // todo: Should we have the Back button here ?
+    // Maybe not b/c it doesn't lead anywhere
```

Comment:
@Gundersanne please comment on this

Reply:
The back button should work? When I try it out on ci.clouddot it goes to the previous step.

Reply:
This is the very first step - "Release" and the Back button here is disabled and IMO it should not even be present.

Reply:
Ah yea we mostly follow the designs here put forth by the UX team

![image](https://user-images.githubusercontent.com/11140201/98352050-d2efbf80-201d-11eb-8237-a4ad1936375d.png)